io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,16 +3,36 @@
   <id type="desktop-application">io.freetubeapp.FreeTube</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>FreeTube</name>
-  <summary>An Open Source YouTube app for privacy</summary>
+
```

Comment:
Please revert to the original formatting. Remove the spacing here.

Reply:
1. The word "open-source" is not a proper noun.
2. The summary should not end with a full stop.